nxdl/nxdl_to_hdf5.py

```diff
@@ -1424,6 +1442,9 @@ def make_class_as_nf_file(clss_nm, dest_dir, symbol_dct={}):
         _string_attr(nf, 'h5py_version', h5py.version.version, do_print=False)
         #_string_attr(nf, 'NEXUS_release_ver', rel_ver)
         entry_grp, entry_nm = get_entry(nf)
+        if entry_grp is None:
+            fatal_error('File does not contain an NXentry group')

```

Comment:
change to `ValueError('File does not contain an NXentry group')`

Reply:
👍